overlord/devicestate/devicestate_test.go

```diff
@@ -801,6 +803,12 @@ func (s *deviceMgrSuite) TestDoRequestSerialErrorsOnNoHost(c *C) {
 		c.Skip("cannot run test when http proxy is in use, the error pattern is different")
 	}
 
+	// check internet access
+	_, err := net.LookupHost("nowhere.nowhere.test")
```

Comment:
That domain name implies we're not actually using the host. Why do we need internet access at all here? Can we use a local IP address (127.*) instead of a domain name for the same effect? @pedronis Any ideas here?

Reply:
It has to actually go out to the DNS server, for example, in the OpenSUSE build jail resolving "localhost" works fine.